### PR TITLE
Feature/governments

### PR DESCRIPTION
**Pull request by pablopareja**

### CODE REVIEW

File: src/app/pages/fields/governments/government-detail.component.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  private countries: Country[];
+  private titleText: String = 'New Government';
+
+  constructor(
+    private auth: AuthService,
+    private countriesService: CountriesService,
+    private router: Router
+  ) {
+    this.countries = new Array<Country>();
```
Comment (clementprdhomme) [inline src/app/pages/fields/governments/government-detail.component.ts]: You don't need to do that, you can just keep:
```js
private countries: Country[] = [];
```

You have done the same for other components.

File: src/app/pages/fields/law/law-detail.component.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+@Component({
+  selector: 'otp-law-detail',
+  templateUrl: './law-detail.component.html',
+  styleUrls: ['./law-detail.component.scss']
+})
+export class LawDetailComponent implements OnInit {
+
+  private countries: Country[];
+  private titleText: String = 'New Law';
```
Comment (clementprdhomme) [inline src/app/pages/fields/law/law-detail.component.ts:15]: Any reason why this is a variable? You have done the same for other components.

File: src/app/pages/fields/governments/government-detail.component.ts (hunk 1 above)
Comment (pablopareja) [reply src/app/pages/fields/governments/government-detail.component.ts]: you're right 👍

File: src/app/pages/fields/law/law-detail.component.ts (hunk 2 above)
Comment (pablopareja) [reply src/app/pages/fields/law/law-detail.component.ts:15]: I was thinking of reusing the same component for editing and thus the text in the header will not always be the same

**Pull request merged**